src/track/track.cpp

```diff
@@ -989,6 +989,63 @@ void Track::shiftCuePositionsMillis(double milliseconds) {
     markDirtyAndUnlock(&locked);
 }
 
+void Track::setHotcueIndicesSortedByPosition(bool compress) {
+    auto locked = lockMutex(&m_qMutex);
```

Comment:
This would be a good candidate for a unit test. wdyt?

Reply:
:+1:

Reply:
Do you have time to implement one?

Reply:
Sure, done.